### PR TITLE
Improved wording

### PR DESCRIPTION
**Pull request by ravage84**
Use same kind of wording in each sub title

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -12,7 +13,7 @@ EXPLAIN WHAT YOU DID, PREFERABLY WITH CODE EXAMPLES, HERE.
 ### What happened
 EXPLAIN WHAT IS ACTUALLY HAPPENING, HERE.
 
-### Expected Behavior
+### What you expected to happen
```
Comment (inoas) [inline .github/ISSUE_TEMPLATE.md]: What I expected to happen

Comment (ravage84) [inline .github/ISSUE_TEMPLATE.md]: Nope: "What you did" -> "What happened" -> "What you expected to happen" makes sense to me.

Comment (inoas) [inline .github/ISSUE_TEMPLATE.md]: Nope, it is a template that is being used by a "user" and he submits the stuff. It is not that the headers will disappear (e.g. those are not Form labels).

What you expected is okay in the moment the Form is up, but afterwards it is wrong.

Comment (inoas) [inline .github/ISSUE_TEMPLATE.md]: Obviously "What I did" then.

**Pull request merged**